tailwind.config.cjs

```diff
@@ -28,7 +28,8 @@ module.exports = {
                 "m-blue-dark": "hsla(220, 59%, 42%, 1)",
                 "m-red": "hsla(343, 92%, 54%, 1)",
                 "m-red-dark": "hsla(343, 92%, 44%, 1)",
-                "sidebar-gray": "hsla(222, 15%, 7%, 1)"
+                "sidebar-gray": "hsla(222, 15%, 7%, 1)",
+                "dialog-black": "rgba(23, 23, 23, 1)"
```

Comment:
the figma should show you which color var to use:

<img width="312" alt="Screenshot 2023-06-26 at 5 28 02 PM" src="https://github.com/MutinyWallet/mutiny-web/assets/543668/44fffb92-49d7-4328-98db-f4904ec65aa5">

Reply:
so grey/800 = m-grey-800

Reply:
sorry, didnt see your comment above this one making the change

Reply:
ah you need to rebase to master to see the new color vars in there